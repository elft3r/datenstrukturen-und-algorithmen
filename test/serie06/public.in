11
10 6 16 20 18 19 17 3 10 8 4
31 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31
15 -1 2 17 -19 18 -19 -30 5 8 -5 13 17 -10 10 -6
17 5 -25 -9 24 25 -30 -18 18 8 -31 10 -6 -33 -4 28 23 -17
26 -18 -30 38 -51 10 18 6 24 39 9 11 50 -52 47 -19 -31 -28 42 -52 -33 -24 -15 50 -10 34 25
30 -1 -51 53 14 -12 46 8 8 -41 -8 -23 39 -43 -40 -58 -52 53 -56 -10 -51 -47 49 -57 28 -8 -14 -48 -10 47 -36
40 66 63 10 77 -15 -69 -14 13 32 -64 52 61 -1 -43 5 -68 4 -41 13 75 48 50 31 -47 17 59 -61 -56 -32 46 15 -57 -53 32 -12 45 77 -64 53 71
50 -35 99 -98 -86 -38 36 -50 28 24 41 -21 -75 -79 52 -57 66 42 71 -43 -71 -56 -55 -13 -88 72 -2 -13 -23 -17 -23 -80 95 -72 8 69 -31 20 69 -7 71 35 12 15 37 -88 -43 -94 -45 -45 -40
60 -22 -70 37 44 -35 19 -78 2 -11 -46 -66 111 83 111 44 -99 -20 44 55 35 25 34 -64 -69 -28 -9 83 1 -93 -55 40 -54 -47 26 -75 -2 39 55 -80 84 -6 -41 65 57 30 77 -58 -99 -55 -87 -79 74 -78 97 -4 -79 88 -41 -67 83
100 -50 88 148 -151 -76 17 49 -112 -15 109 -174 -28 -84 -65 138 8 -198 -151 -146 -160 162 87 -119 75 -131 -69 47 -48 169 150 110 147 -84 15 -48 180 -187 -189 13 133 142 57 190 38 173 -160 -52 -17 106 8 -89 185 -196 106 -126 121 -4 149 172 10 -101 182 -199 -186 -81 99 -177 -159 72 68 -115 114 161 -146 85 174 -125 30 75 34 3 110 -27 -67 59 71 -174 -55 197 -173 -25 -172 -156 77 132 47 -115 88 150 34
120 -21 96 -19 65 105 105 -107 88 183 83 179 -18 128 39 -171 89 -6 -200 212 70 4 -112 -83 91 46 -106 106 203 178 217 -224 -236 224 124 -46 -7 -41 62 50 119 48 4 -57 -25 148 -154 -21 -49 14 211 -85 -156 -197 33 -142 -31 -34 181 -97 189 176 235 -206 -213 149 -93 -218 -13 -151 136 -200 9 -103 190 -123 -217 46 103 18 -20 227 -77 -27 208 184 -158 -175 -157 128 118 207 225 117 -6 -57 -12 172 105 -178 -218 226 -196 201 128 85 -204 -176 -188 111 -192 -178 196 -3 203 -155 -98 63 166 -146 52
