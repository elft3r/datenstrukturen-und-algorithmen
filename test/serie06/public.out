18 8 8 10
3 8 13 18 23 28 31 18 16
2 -5 10 2 2
5 -18 -4 -17 -17 -4
-18 18 11 -31 -10 25 -10 6
-1 8 -40 -51 -8 -14 -14 -12
63 -14 5 4 31 -32 -12 53 4 13
-38 28 -57 42 -55 -17 8 20 15 -45 -17 -21
-22 -11 83 35 -28 -9 -47 39 30 -58 -4 -41 -11 -6
-50 17 -65 -151 75 47 15 13 142 -17 -89 121 -101 68 85 34 3 -55 -25 47 13 10
65 88 39 70 4 178 -46 50 4 -21 -142 -31 149 -93 -103 18 184 -157 117 -12 128 -188 -3 52 4 4
